src/lib/components/popup/CloseConfirmationPopup.svelte

```diff
@@ -0,0 +1,63 @@
+<script lang="ts">
+  import { ModalWrapperV1 } from "..";
+  import Button from "../buttons/Button.svelte";
+
+  export let isOpen = false;
+  export let isSaveDisable = false;
+  export let onChangeModalState: (flag: boolean) => void;
```

Comment:
The convention is always that the activity like "changed", "clicked", "moved", "dropped", should be at the end.

So rather than `onChangeModalState`, the name should be `onModalStateChanged`

Reply:
Done